packages/bruno-app/src/components/ResponsePane/QueryResult/index.js

```diff
@@ -35,15 +42,14 @@ const formatResponse = (data, mode, filter) => {
     if (typeof parsed === 'string') {
       return parsed;
     }
-
     return safeStringifyJSON(parsed, true);
   }
 
   if (typeof data === 'string') {
     return data;
   }
-
-  return safeStringifyJSON(data);
+  //return safeStringifyJSON(data)
```

Comment:
I'd suggest to remove commented code.
```suggestion
  //return safeStringifyJSON(data)
```